src/EFTools/EntityDesignDataSourceWizardExtension/EntityDesignDataSourceWizardExtension.csproj

```diff
@@ -40,6 +24,26 @@
   <PropertyGroup>
     <DelaySign>true</DelaySign>
   </PropertyGroup>
+  <PropertyGroup Condition="'$(Configuration)|$(Platform)' == 'Debug|x64'">
```

Comment:
This project and three other projects apparently have a dependency on the `Microsoft.VisualStudio.Modeling.Sdk.Diagrams.GraphObject` dll, which looks to be only supported for x64 architecture.

To resolve the issue, I used the Configuration Manager within VS to update each project's supported architectures, although this was my first time using that tool so I'm hoping it's correct.

![image](https://github.com/dotnet/ef6tools/assets/16968319/6322d9d1-f489-40b5-a774-38d19bdc7beb)

Reply:
@phil-allen-msft this is the only non-test project that is being updated to be x64 exclusive. I'm not 100% sure about the full function of this project - does this change seem appropriate to you?

Reply:
I reverted this change since it seems to break other areas.